src/components/QRShare/index.js

```diff
@@ -72,7 +66,7 @@ class QRShare extends Component {
                     family="EXP_NEW_KANSAS_MEDIUM"
                     fontSize={22}
```

Comment:
NAB as it's not added here, should we hard code values here for `fontSize`?

Reply:
Yeah good point, I think we can reuse [`fontSizeXLarge`](https://github.com/Expensify/App/blob/2417af0e51dc64ecb1623e4157a9e2b394f29776/src/styles/variables.js#L53) in order to avoid hardcoding a style attribute